tools/sdk-readme-table-generator/TableGenerator/Program.fs

```diff
@@ -8,19 +8,19 @@ open TableGenerator.Table
 let inputBranches =
         [ { GitBranchName = "master"
             DisplayName = "Master<br>(6.0.x&nbsp;Runtime)"
-            AkaMsChannel = Some("net6/dev") }
+            AkaMsChannel = Some("6.0/daily") }
           { GitBranchName = "release/6.0.1xx-preview1"
             DisplayName = "Release/6.0.1xx-preview1<br>(6.0.x&nbsp;Runtime)"
-            AkaMsChannel = Some("net6/preview1") }
+            AkaMsChannel = Some("6.0-preview1/daily") }
           { GitBranchName = "release/5.0.3xx"
             DisplayName = "Release/5.0.3XX<br>(5.0 Runtime)"
-            AkaMsChannel = Some("net5/5.0.3xx/daily") }
+            AkaMsChannel = Some("5.0.3xx/daily") }
           { GitBranchName = "release/5.0.2xx"
             DisplayName = "Release/5.0.2XX<br>(5.0 Runtime)"
-            AkaMsChannel = Some("net5/5.0.2xx/daily") }
+            AkaMsChannel = Some("5.0.2xx/daily") }
           { GitBranchName = "release/5.0.1xx-rtm"
             DisplayName = "5.0.100 RTM<br>(5.0 Runtime)"
-            AkaMsChannel = Some("net5/5.0.1xx/daily") }
+            AkaMsChannel = Some("5.0.1xx/daily") }
```

Comment:
And this one: https://aka.ms/dotnet/5.0.1xx/daily/Sdk/dotnet-sdk-win-x64.exe

Reply:
I’m not sure what’s happening with 501. Before making the change, I checked the original link format and it also didn’t seem to work, but I’ll look into it.

Reply:
Still not sure what is going on here. I don't have any opinion about if it should remain the original link or the new link, but neither work, so which is correct?